Assets/StreamingAssets/Data/Furniture.xml

```diff
@@ -453,4 +453,24 @@
     <LocalizationCode>furn_accumulator</LocalizationCode>
     <UnlocalizedDescription>furn_accumulator_desc</UnlocalizedDescription>
   </Furniture>
+  
+  <Furniture objectType="berth">
+    <Name>Ship Berth</Name>
+    <MovementCost>1</MovementCost>
+    <PathfindingModifier>1</PathfindingModifier>
+    <PathfindingWeight>0</PathfindingWeight>
+    <Width>3</Width>
+    <Height>1</Height>
+
+    <BuildingJob jobTime="1" />
+
+    <Params>
+      <Param name="occupied" value="0" />
```

Comment:
In your .xml files changes , some use 2 spaces indent some are 4 spaces and some are tabs. You should probably pick one